src/index.html

```diff
@@ -1,13 +1,74 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card__container" data-qa="card">
+      <img src="/images/imac.jpeg" alt="imac"
+      class="card__image"
+      >
+      <div class="description__block">
+        <p class="title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+        <p class="product__code">
+          Код товара: 195434
+        </p>
+        <div class="feedback">
+          <div class="stars__container">
+            <span class="stars__item">
+              <img src="/images/star-active.svg" alt="star-active"
+              class="star"
+              >
+            </span>
+            <span class="stars__item">
+              <img src="/images/star-active.svg" alt="star-active"
+              class="star"
+              >
+            </span>
+            <span class="stars__item">
+              <img src="/images/star-active.svg" alt="star-active"
+              class="star"
+              >
+            </span>
+            <span class="stars__item">
+              <img src="/images/star-active.svg" alt="star-active"
+              class="star"
+              >
+            </span>
+            <span class="stars__item">
+              <img src="/images/star.svg" alt="star-active"
+              class="star"
+              >
+            </span>
```

Comment:
use your code from previous task. stars should be added to the page as background, not as images